### PR TITLE
cri-o: rename default branch to main

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
/hold

because the rename hasn't happened yet

File: ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-main-presubmits.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ presubmits:
   - agent: jenkins
     always_run: true
     branches:
-    - master
-    - release-1.21
+    - main
+    - release-1.2
```
Comment (fidencio) [inline ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-main-presubmits.yaml:7]: Nice! :-)

**Review by fgiudici — APPROVED** [review]
LGTM!

**Pull request merged**